docs/developer/typescript/01_gettingStarted/01_hello/01_ecs.md

```diff
@@ -0,0 +1,61 @@
+# The ECS Pattern
+The [Entity Component System](https://en.wikipedia.org/wiki/Entity_component_system) is a pattern used to organize our code when writing software.  
+In this pattern:
+- Logic is represented as `System`s, and they define the behavior of the application.
+- Data is represented as `Component`s that have no behavior or identifiers attached to them.
+- Components are attached to `Entities`.
+- An `Entity` is essentially a "name" that can be used to connect a group of components into a single "thing".
+
+## Creating an Entity
+```ts
+const entity = ECS.createEntity()
+```
+
+## Adding Components
+Ethereal Engine requires a specific set of Components in order to make an Entity show up on the screen:
+- **VisibleComponent**
+- **TransformComponent**
+- **PrimitiveGeometryComponent** or **MeshComponent**
```

Comment:
I believe there might be more ways to get geometry onto an entity. Like more complex components. But it might be confusing. I'd like to hear what you think @HexaField

Reply:
primitive geom is nice and simple. the point of this tutorial is to demonstrate the ecs. we also are still defining what these mesh/geom components are - that is in a lot of flux and will be until our webgpu impl is well under way

Reply:
I was told these two were the only methods. Will mark this one as solved, but let me know if I should add any other methods here.